.github/workflows/create-next-milestone.yml

```diff
@@ -9,11 +9,11 @@ jobs:
     steps:
       - name: Get next minor version
         id: semvers
-        uses: WyriHaximus/github-action-next-semvers@0.1.0
+        uses: WyriHaximus/github-action-next-semvers@b135abb108d66990a85e18623d906404f4350ce4
```

Comment:
This is actually not the latest version -- looking at https://github.com/WyriHaximus/github-action-next-semvers/tags this seems to be `v1.0`, whereas `v1.1.0` is out. Was this intended?

Reply:
I tried that, but it seems like the version that matters to GitHub is the one that's published here: https://github.com/marketplace/actions/next-semvers, which is `v1.0`.
Trying `v1.1.0` fails to find the version.

Reply:
Thanks for clarifying that this was intended; looks good to me.

Ahaha I find it somewhat funny that if you press `Use latest version` on that page you don't get the commit hash as recommended by the security team. A classic!